### PR TITLE
[Feature](bangc-ops): update proto for poly_nms.

### PR DESCRIPTION
**Pull request by guzhonghao**

### CODE REVIEW

**Review by mahxn0 — APPROVED** [review]
LGTM

File: mlu_op_test_proto/mlu_op_test.proto [hunk 1]
```diff
@@ -118,6 +118,8 @@ message Node {
   optional DivParam    div_param                      = 4001;   // param
   optional LogParam    log_param                      = 4002;     // param
   optional SqrtParam   sqrt_param                     = 4003;  // param
+  optional PsRoiPoolForwardParam psroipool_forward_param = 4004; //param
+  optional PolyNmsParam poly_nms_param = 1244;
```
[the remark this answers is not in the document]
Comment (guzhonghao) [reply mlu_op_test_proto/mlu_op_test.proto]: 已修改

**Review by DanieeelLiu — APPROVED** [review]
LGTM

**Review by wushaoqiang2168 — APPROVED** [review]
LGTM

**Review by DanieeelLiu — APPROVED** [review]

**Review by WangYuan-cn — APPROVED** [review]

**Pull request merged**